### DISCUSSION TITLE
How to understand this requests per second graph?

### DISCUSSION THREAD

**Discussion by 64J0** [coroot/coroot#202] [Q&A] [answered]
Hey, hope you're good. I'd like to know how to interpret this requests per second graph:

![image](https://github.com/coroot/coroot/assets/50725287/3b9d3676-3f80-48ef-b008-1d4cd212ccd8)

What does the `m` suffix stands for there?

I'm using coroot Helm chart version `coroot-0.8.14`.

cc @leomaiadatarisk

---

Some references we were taking a look to try to understand:

- https://github.com/coroot/coroot/blob/main/auditor/slo.go#L164
- https://github.com/coroot/coroot/blob/main/front/src/components/Chart.vue#L73

**Comment by apetruhin** [comment, marked as answer]
@64J0,

'150m' means '150 [milli](https://en.wikipedia.org/wiki/Milli-) requests per second' or '0.15 requests per second' or ’60*0.15 = 9 requests per minute'

**Reply by 64J0** [reply]
Got it, thanks for the clarification @apetruhin.
